File: JanperCab/5-JanperCab/ClientApp/src/app/_models/duraform-misc-price/DuraformMiscPriceHeatStripDto.ts
import { HeatStripSizeEnum } from './../../_enums/HeatStripSizeEnum';
import { DuraformMiscPriceDto } from './DuraformMiscPriceDto';

export class DuraformMiscPriceHeatStripDto extends DuraformMiscPriceDto {
  heatStripSize: HeatStripSizeEnum;

  constructor() {
    super();
    this.$type =
      '_3_Application.Dtos.DuraformMiscPrice.DuraformMiscPriceHeatStripDto, 3-Application';
  }
}

File: JanperCab/3-Application/Dtos/DuraformMiscPrice/DuraformMiscPriceHeatStripDto.cs
using _1_Domain;
using _1_Domain.Enum;

namespace _3_Application.Dtos.DuraformMiscPrice
{
    public class DuraformMiscPriceHeatStripDto : DuraformMiscPriceDto
    {
        public HeatStripSizeEnum HeatStripSize { get; set; }
    }
}